test/e2e/network/dns.go

```diff
@@ -380,7 +380,12 @@ var _ = SIGDescribe("DNS", func() {
 		validateTargetedProbeOutput(f, pod3, []string{wheezyFileName, jessieFileName}, svc.Spec.ClusterIP)
 	})
 
-	ginkgo.It("should support configurable pod DNS nameservers", func() {
+	/*
+		Release : v1.16
```

Comment:
Should we change this to 1.17?
This won't go into 1.16 because of code freeze https://github.com/kubernetes/sig-release/tree/master/releases/release-1.16#code-freeze

Also, is there a chance to squash the two commits into one?

Other than that, the test seems to be good to go https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-default&include-filter-by-regex=should%20support%20configurable%20pod%20DNS%20nameservers&graph-metrics=test-duration-minutes

Reply:
Thanks for the observation, the release version has been updated and the commits have been squashed in cd874bd0d12efb617644d27d606cd5f1b8fe273d.